content/applications/sales/sales/invoicing/down_payment.rst

```diff
@@ -9,176 +9,268 @@ With a down payment, the buyer pays a portion of the total amount owed while agr
 remaining amount at a later date. In turn, the seller provides goods or services to the buyer after
 accepting the down payment, trusting that the remaining amount will be paid later on.
 
-Initial down payment request
-============================
+Create invoices
+===============
 
 When a sales order is confirmed, the option to create an invoice becomes available, via the
 :guilabel:`Create Invoice` button, located in the upper-left corner of the sales order form. When
 clicked, a :guilabel:`Create invoices` pop-up appears.
 
+.. image:: down_payment/create-invoices-popup-form.png
+   :align: center
+   :alt: Create invoices pop-up form that appears in Odoo Sales.
+
 .. note::
-   Invoices are automatically created in drafts, so they can be reviewed before validation.
+   Invoices are automatically created as drafts, so they can be reviewed before validation.
 
 On the :guilabel:`Create invoices` pop-up, there are 3 options to choose from in the
 :guilabel:`Create Invoice` field:
 
 - :guilabel:`Regular invoice`
-   .. note::
-      If :guilabel:`Regular Invoice` is selected the other fields disappear, as they only pertain to
-      down payment configurations.
 - :guilabel:`Down payment (percentage)`
 - :guilabel:`Down payment (fixed amount)`
```

Comment:
This isn't paramount either, but it might be nice to have a short description/definition of each of these 3 choices

Reply:
This, too, may be unnecessary. In my opinion, these elements seem fairly self-explanatory. So, for the sake of staying as concise as possible, I'll leave these elements as-is -- but if other reviewers bring this up, I'll make the necessary changes right away. Thanks! 👍